pkg/k8sclient/fake/clients.go

```diff
@@ -119,6 +119,10 @@ func (c *Clients) CtrlClient() client.Client {
 	return c.ctrlClient
 }
 
+func (c *Clients) CtrlCache() client.Reader {
```

Comment:
I'd suggest adding the internal workings you wrote in the PR description in the GoDoc here.

Reply:
Makes sense, though you probably mean `pkg/k8sclient/clients.go` ?

Reply:
Yes